HelpSource/Classes/ToolBar.schelp

```diff
@@ -0,0 +1,118 @@
+TITLE:: ToolBar
+summary:: An inline button bar populated by MenuActions
+categories:: GUI
+related:: Classes/Menu, Classes/MenuAction, Classes/ToolBar, Classes/MainMenu
+
+DESCRIPTION::
+A ToolBar is similar to a link::Classes/Menu::, with several key differences:
+LIST::
+	## ToolBars act as regular link::Classes/View::s, and can be placed in layouts or in other views.
+	## MenuActions on a toolbar appear as buttons instead of being listed menu-style
+	## By default, MenuActions that have icons show ONLY the icon. MenuActions with text will appear as a text button. This can be changed using the toolButtonStyle property.
+::
+
+code::
+(
+View().fixedWidth_(300).layout_(HLayout(
+	ToolBar(
+		MenuAction("Option A", { "Option A".postln }),
+		MenuAction("Option B", { "Option B".postln }),
+		MenuAction("Option C", { "Option C".postln })
+	)
+)).front
+)
+::
+
+CLASSMETHODS::
+
+METHOD:: new
+	Create a new ToolBar containing one or more actions.
+
+	ARGUMENT::
+		One or more link::Classes/MenuAction::
+
+INSTANCEMETHODS::
+
+METHOD:: orientation
+	The horizontal or vertical orientation of the ToolBar
+	ARGUMENT::
+		A link::Classes/QOrientation::
+
+METHOD:: toolButtonStyle
+	Controls whether the toolbar shows icons, text, or both.
+	ARGUMENT::
+		A link::Classes/QToolButtonStyle::
+	DISCUSSION::
+		CODE::
+(
+~icon = Image(64).draw({ Pen.fillOval(Rect(0, 0, 64, 64)) });
+~styles = QToolButtonStyle.classVarNames;
```

Comment:
This class is not defined.

Reply:
Added it to `enums.sc`; @scztt you may want to review

Reply:
lgtm